jdk/README.md

```diff
@@ -0,0 +1,162 @@
+## Using Red Hat Single Sign-On 7.5 container images with alternative JDK
+
+The Red Hat Single Sign-On 7.5 for OpenJDK container images use the [Red Hat OpenJDK 11](https://access.redhat.com/documentation/en-us/openjdk/11/html-single/getting_started_with_openjdk_11/index#openjdk-overview), a free and open source implementation of the Java Platform, Standard Edition (Java SE) by default.
+
+To provide an illustrative example on how to install an alternative JDK and instruct the Red Hat Single Sign-On 7.5 container images to use it, the 'ibm-semeru-open-11-jdk' subdirectory contains a definition of Red Hat Single Sign-On 7.5 container file switching the JDK runtime to the latest available release of [IBM Semeru Runtime Open Edition Java 11 (LTS)](https://github.com/ibmruntimes/semeru11-binaries/releases/latest).
+
+## Building the Red Hat Single Sign-On 7.5 container image with latest release of IBM Semeru 11 JDK Open Edition
+
+You can build this example by creating a new build using OpenShift CLI (oc) tool. Specify:
+* The vanilla `rh-sso-7/sso75-openshift-rhel8` image stream to use as the builder,
+* The Docker build strategy to use for build execution, and
+* The Git repository, branch name and particular context directory within the repository as the source of the build
+
+**Prerequisites**
+
+* Ensure you have [OpenShift CLI (oc) installed](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli)
+
+**Procedure**
+
+1. Ensure that you are logged in as a cluster administrator or a user with project administrator access to the global `openshift` project. Choose the following command based on your version of OpenShift Container Platform:
+
+   * If you are running an OpenShift Container Platform v3 based cluster instance on (some) of your master host(s), perform the following:
+
+      ```
+      $ oc login -u system:admin
+      ```
+
+   * If you are running an OpenShift Container Platform v4 based cluster instance, [log in to the CLI](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands) as the [kubeadmin](https://docs.openshift.com/container-platform/latest/authentication/remove-kubeadmin.html#understanding-kubeadmin_removing-kubeadmin) user:
+
+      ```
+      $ oc login -u kubeadmin -p password https://openshift.example.com:6443
+      ```
+
+2. Execute the following command:
+
+   ```
+   $ oc new-build \
+     --context-dir=jdk/ibm-semeru-open-11-jdk \
+     --image-stream=sso75-openshift-rhel8 \
+     --name=sso75-openshift-rhel8-ibm-semeru-11-jdk \
+     --namespace=openshift \
+     --strategy=docker \
+     https://github.com/jboss-container-images/redhat-sso-7-openshift-image.git#sso75-dev
```

Comment:
@dale-fu @levicic JFTR, obviously, till the PR is merged, you will need to change the last line in the above command to point to my fork & corresponding branch instead of the form above. IOW, create new build as follows:

```
$ oc new-build \
     --context-dir=jdk/ibm-semeru-open-11-jdk \
     --image-stream=sso75-openshift-rhel8 \
     --name=sso75-openshift-rhel8-ibm-semeru-11-jdk \
     --namespace=openshift \
     --strategy=docker \
     https://github.com/iankko/redhat-sso-7-openshift-image.git#semeru
```

The rest of the proposed steps should work without additional changes AFAICT.